buildenv/jenkins/JenkinsfileBase

```diff
@@ -511,6 +511,8 @@ def setup() {
 		VENDOR_TEST_BRANCHES = (params.VENDOR_TEST_BRANCHES) ? "--vendor_branches \"${params.VENDOR_TEST_BRANCHES}\"" : ""
 		VENDOR_TEST_DIRS = (params.VENDOR_TEST_DIRS) ? "--vendor_dirs \"${params.VENDOR_TEST_DIRS}\"" : ""
 		VENDOR_TEST_SHAS = (params.VENDOR_TEST_SHAS) ? "--vendor_shas \"${params.VENDOR_TEST_SHAS}\"" : ""
+		env.IS_SVT_TESTREPO = params.VENDOR_TEST_REPOS.contains('SVTTestRepo') ? true : false
```

Comment:
This may fail if `params.VENDOR_TEST_REPOS` is not set.

Reply:
https://github.com/adoptium/aqa-tests/pull/5418#issuecomment-2225741481 passed because params.VENDOR_TEST_REPOS is set to empty, not undefined. I think we should do 
```
env.IS_SVT_TESTREPO = (params.VENDOR_TEST_REPOS  && params.VENDOR_TEST_REPOS.contains('SVTTestRepo')) ? true : false
```